src/geo/crs/CRS.js

```diff
@@ -83,22 +84,31 @@ export var CRS = {
 		return new Bounds(min, max);
 	},
 
-	// @method distance(latlng1: LatLng, latlng2: LatLng): Number
-	// Returns the distance between two geographical coordinates.
-
 	// @property code: String
 	// Standard code name of the CRS passed into WMS services (e.g. `'EPSG:3857'`)
-	//
+
 	// @property wrapLng: Number[]
 	// An array of two numbers defining whether the longitude (horizontal) coordinate
 	// axis wraps around a given range and how. Defaults to `[-180, 180]` in most
 	// geographical CRSs. If `undefined`, the longitude axis does not wrap around.
-	//
+	set wrapLng(wrapLng) {
+		Wrap.lng = wrapLng;
+	},
+	get wrapLng() {
+		return Wrap.lng;
+	},
+

```

Comment:
- We can't use getters/setters yet because we support old versions of IE — this would be a breaking change.
- I'm really confused by this global state setup with `Wrap`, can you explain what's going on here?

Reply:
Apologies the intent was not to make the code more confusing.  Especially if there are incompatibilities Introduced.  
Re wrap. Ithe two wraped values were externalised to break the circular dep between LatLng and CRS.

Feel free to close the PR if it serves no value.

Reply:
Sorry, I may have sounded harsh but I'm just trying to understand, not criticizing. If there's a legitimate case where circular deps cause problems without an easier workaround, we should definitely continue looking into this.

Reply:
@mourner, originally I recall having errors with the Karma breaking because of the circular dependencies when calling the test runner with tests we have in a project (react/typescript/leaflet).  That was over 12 months ago, this project uses karma without error, and the former project has moved to jest.  So I do not have a hard requirement to fix this anymore and cannot produce a repro now.  

Further, as you mention, breaking changes for IE because of the use of the setters/getters, this was originally meant to be a refactor with minimal surface changes, I could not achieve this.   So I believe this should be closed to avoid introducing more pain that it is trying to solve.